Tests/TransifexTests/TransifexTests.swift

```diff
@@ -330,7 +330,7 @@ final class TransifexTests: XCTestCase {
         
         XCTAssertNil(cache.get(key: "key1", localeCode: "en"))
         XCTAssertNotNil(cache.get(key: "key2", localeCode: "en"))
-        XCTAssertNil(cache.get(key: "key3", localeCode: "en"))
+        XCTAssertNotNil(cache.get(key: "key3", localeCode: "en"))
```

Comment:
I think we should update all tests to include all cases described in the nice table you created.

Reply:
I have pushed a commit that replaces the existing policy unit tests with two tests that use the provided table.